lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -0,0 +1,187 @@
+define((require, exports, module) ->
+  Behaviour = require("ace/mode/behaviour").Behaviour
+
+  isCommentToken = (token) -> token? and /comment/.test(token.type)
```

Comment:
@yanenkoa why would you expect them to be faster? It is [nearly twice slower](https://jsperf.com/substring-test) unless it is really cached (no idea if JS engine can or cannot cache literals automatically, and from the results of test above I can't judge if it does or does not). 

But I don't think that it matters. 

I would expect regexp to be used when you really have a regexp, and when it is just a  substring, I automatically start thinking if I am missing anything. 

> this is how other behaviors are written

Do you mean those in ACE code?